kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Serialization.java

```diff
@@ -46,6 +48,7 @@ private Serialization() { }
     JSON_MAPPER.registerModule(new JavaTimeModule());
   }
   private static final ObjectMapper YAML_MAPPER = new ObjectMapper(new YAMLFactory());
+  private static final Yaml SNAKE_YAML_MAPPER = new Yaml();
```

Comment:
This class (`Yaml`) is not thread safe, should be instantiated for each request.

nit: it's not a mapper either.

The field should be removed. See proposed changes in lines 62-64 for the thread-safety issue.